redeem/gcodes/M2x.py

```diff
@@ -288,7 +288,7 @@ def process_gcode(self, g):
       file_g = Gcode({"message": line})
       self.printer.processor.enqueue(file_g)
     if self.printer.sd_card_manager.get_status():
-      logging.info("M24: file complete")
+      logging.info("M24: Print complete")
```

Comment:
Wouldn't it be more appropriate to say `Print from file complete`?

Reply:
Sounds good to me. Prepare to squash the change into one commit.